.github/CODEOWNERS

```diff
@@ -302,6 +302,9 @@ sdk/ai/ai-inference-rest @glharper @dargilco @jhakulin
 # ServiceLabel: %Maps
 /sdk/maps/ @dubiety @andykao1213
 
+# Cosmos API review files
+/sdk/cosmosdb/cosmos/review/*api.md @amanrao23 @aditishree1 @sajeetharan @topshot99
```

Comment:
More specific entry should be after generic ones.  Could you move this entry to be after the management one?

Reply:
Sorry I didn't make it clearer.  I suggested to move this after the following entry at Line 311

```
# Managment API review files
/sdk/**/review/arm-*.api.md @qiaozha @MaryGao @Azure/azure-sdk-for-js-core
```